src/modules/block/components/blocks/blocks.js

```diff
@@ -22,41 +22,48 @@ export const defaultFilters = {
 };
 
 // eslint-disable-next-line max-statements
-const Blocks = ({ filters, applyFilters, clearFilter, clearAllFilters, sort, changeSort }) => {
+const Blocks = () => {
```

Comment:
Changes here probably introduces regression, in that filters are not cleared. 
1. Add height filter (or any other filter)
2. Remove height filter by clicking x
3. Expected: Height field in filter dropdown should be cleared

(Same for remove all filters)

Reply:
It seems like a general problem as the same behaviour exists in all the other places where the hook filter is used. We'll need to create a ticket to fix them all

Reply:
This problem doesnt exists in the `release/3.0.0` branch though, feels strange to introduce a new bug. 

Problem with this behaviour is that after cleaning and values are still in the dropdown the apply button does nothing.

Reply:
It exists in the other pages like Transactions monitor that use the useFilter hook. That's what I was referring to as a general problem